rules/windows/sysmon/silenttrinity_stager_msbuild_activity.yml

```diff
@@ -0,0 +1,26 @@
+title: Silenttrinity Stager Msbuild Activity
+id: 50e54b8d-ad73-43f8-96a1-5191685b17a4
+description: Detects a possible remote connections to Silenttrinity c2
+references:
+    - https://www.blackhillsinfosec.com/my-first-joyride-with-silenttrinity/
+tags:
+    - T1127.001
+    - TA0002
+    - T1127
+    - attack.defense_evasion
+status: experimental
+author: Kiran kumar s, oscd.community
+date: 2020/10/11
+logsource:
+    category: sysmon
+    product: windows
+detection:
+    selection:
+        EventID: 3
+        ParentImage|endswith: '*\msbuild.exe'
```

Comment:
could you please remove `*` from `'*\msbuild.exe'` as it is already supposed by the `endswith`?

Reply:
Resolved

Reply:
It seems that it wasn't resolved %) Could you please apply the recommendation?

Reply:
Hello @skirankumar! It seems that it wasn't resolved %) Could you please apply the recommendation?

Reply:
Yeah I removed the event ID then the checks failed ! any suggestions ?

Reply:
I have made Few new changes 
1) Adding Dstport field 
2) Removed * from '*\msbuild.exe' 
The test have failed @yugoslavskiy  is it due to the bug you mentioned that all people are facing or any formatting issue ?kindly let me know if nay changes required.